src/neo/Network/P2P/TaskManager.cs

```diff
@@ -25,6 +25,9 @@ private class Timer { }
 
         private readonly NeoSystem system;
         private const int MaxConncurrentTasks = 3;
+        /// <summary>
+        /// The hashes corresponding to the completed tasks.
```

Comment:
This is not for task, is more for inventories, or payloads already received

Reply:
Updated